src/main/java/de/plushnikov/intellij/plugin/activity/LombokProjectValidatorActivity.java

```diff
@@ -79,29 +81,47 @@ public void runActivity(@NotNull Project project) {
     if (hasLombokLibrary && !annotationProcessorsEnabled &&
       ProjectSettings.isEnabled(project, ProjectSettings.IS_ANNOTATION_PROCESSING_CHECK_ENABLED, true)) {
 
-      String annotationProcessorsConfigName = new AnnotationProcessorsConfigurable(project).getDisplayName();
+      suggestEnableAnnotations(project, group);
+    }
+  }
 
-      Notification notification = group.createNotification(LombokBundle.message("config.warn.annotation-processing.disabled.title"),
-        LombokBundle.message("config.warn.annotation-processing.disabled.message", project.getName()),
-        NotificationType.ERROR,
-        new SettingsOpeningListener(project, annotationProcessorsConfigName));
+  private void suggestEnableAnnotations(Project project, NotificationGroup group) {
+    String annotationProcessorsConfigName = new AnnotationProcessorsConfigurable(project).getDisplayName();
 
-      Notifications.Bus.notify(notification, project);
-    }
+    Notification notification = group.createNotification(LombokBundle.message("config.warn.annotation-processing.disabled.title"),
+      LombokBundle.message("config.warn.annotation-processing.disabled.message", project.getName()),
+      NotificationType.ERROR,
+      (not, e) -> {
+        if (e.getEventType() == HyperlinkEvent.EventType.ACTIVATED) {
+          enableAnnotations(project);
```

Comment:
I think we can/should make additionally something like `not.expire();` here

Reply:
is that necessary? Notification exprires even without calling this method.

Reply:
I just tested it, and I think if looks as good additional feedback, if notification expires after click on "Enable"-Link.
Without expire you can click multiple times on it.

Reply:
great :+1:  I understand it now

Reply:
fixed